### PR TITLE
DEVEXP-388: Adding note on S3 bucket security configuration during UPI.

### PR DESCRIPTION
**Pull request by ricardomaraschini**
Adds a WARNING refering user to AWS documentation regarding S3 bucket security configuration. During IPI these security measures are automatically taken.

### CODE REVIEW

File: modules/registry-configuring-storage-aws-user-infra.adoc [hunk 1]
```diff
@@ -36,3 +36,11 @@ storage:
     bucket: <bucket-name>
     region: <region-name>
 ----
+
+[WARNING]
+====
```
Comment (bmcelvee) [inline modules/registry-configuring-storage-aws-user-infra.adoc]: I would suggest removing the link to the blog, and simplifying the warning as follows: 

[WARNING]
====
To secure your registry images in AWS, link:https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-publicaccessblockconfiguration.html[block public access]
to the S3 bucket. 
====

**Comment by bmcelvee** [comment]
Thanks, @ricardomaraschini! I left a suggestion in-line. According to the Jira, this looks like it's only applicable to 4.2. Is that correct?

File: modules/registry-configuring-storage-aws-user-infra.adoc (hunk 1 above)
Comment (ricardomaraschini) [reply modules/registry-configuring-storage-aws-user-infra.adoc]: Done.

**Comment by ricardomaraschini** [comment]
@bmcelvee I would say it applies to all versions. Let me check and i will get back to you.

**Comment by ricardomaraschini** [comment]
@bmcelvee Just 4.2, though not bad to add to 4.1 docs as well as lots of customers set up the registry on their own.

**Comment by bmcelvee** [comment]
Thank you! I'll go ahead and merge + cherrypick to 4.1 and 4.2.

This is a minor change (addition of a warning) and should not require QE review.

**Pull request merged**